vendor/k8s.io/kubernetes/staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -111,6 +116,7 @@ func NewCustomResourceDefinitionHandler(
 		delegate:                delegate,
 		restOptionsGetter:       restOptionsGetter,
 		admission:               admission,
+		maxRequestBodyBytes:     maxRequestBodyBytes,
```

Comment:
missing the use of this further down in the request scope https://github.com/openshift/origin/pull/23987/commits/5f027c5c20e3a8fdc3cc7acf92a46bd47bc4cdea#diff-b4feb82ec380748d1f9374cbf25bfc7bL372

Reply:
Compare https://github.com/kubernetes/kubernetes/pull/83436/commits/a4e734ac9fea9affdffee9ac845c15e1abacfab3#diff-2f720a7e61a4ddfcfe9a9e353be8784bR594 in upstream.

Reply:
To address this I needed to essentially cherry pick " UPSTREAM: 73805: Adding a limit on the size of request body the apiserver will decode for write operations" which was added in 3.11 and not yet back-ported.

Reply:
Why was it left out in the first place?

Reply:
added below